tagstore/backends/tagstore_django/models.py

```diff
@@ -0,0 +1,78 @@
+"""
+Tagstore backend that uses the django ORM
+"""
+from django.db import models
+
+MAX_CHAR_FIELD_LENGTH = 180
+
+
+class Taxonomy(models.Model):
+    """
+    A taxonomy is a collection of tags, some of which may be organized into
+    a hierarchy.
+    """
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=MAX_CHAR_FIELD_LENGTH)
+    owner_id = models.BigIntegerField(null=False)
```

Comment:
@bradenmacdonald : Default value for null is `False`. Is there any specific reason why are we adding this?